File: kaggle_environments/envs/kore_fleets/starter_bots/java/test/PointTest.java
package test;

import org.junit.Assert;
import org.junit.Test;

import kore.Point;

public class PointTest {

    @Test
    public void fromIndexToIndex_isIdentity() {
        int idx = 254;
        int size = 31;

        Point point = Point.fromIndex(idx, size);
        int mirroredIdx = point.toIndex(size);

        Assert.assertEquals(idx, mirroredIdx);
    }
    
}

File: kaggle_environments/envs/kore_fleets/starter_bots/ts/test/PointTest.ts
import { describe } from 'mocha';
import { expect } from 'chai';

import { Point } from "../kore/Point";

describe('Point', () =>  {
    it('fontIndex toIndex isIdetity', () => {
        const idx = 254;
        const size = 31;

        const point = Point.fromIndex(idx, size);
        const mirroredIdx = point.toIndex(size);

        expect(mirroredIdx).to.equal(idx);
    })

});
